src/Core/Injector/InjectorLoader.php

```diff
@@ -42,7 +44,7 @@ public function getManifest()
             );
         }
         if (empty($this->manifests)) {
-            throw new BadMethodCallException("No injector manifests available");
+            throw new BadMethodCallException(self::NO_MANIFESTS_AVAILABLE);
```

Comment:
Seems like this would be a good place to throw a custom exception. Maybe, we can look at doing something like in CMS5/CMS6.